wordssite/words/migrations/0002_choice_word_word_translation_delete_translation_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.0.4 on 2022-05-03 14:49
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('words', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Choice',
```

Comment:
What is this model for?

Reply:
I was thinking that it is mandatory for some reason, I will delete it